File: team-5-definitely-colins-team/7-even-or-odd-which-is-greater.js
function evenOrOdd(str) {
  let evens = 0
  let odds = 0
  for (const digit of str) {
    digit % 2 === 0 ? evens += +digit : odds += +digit 
  }
  if (odds > evens) return "Odd is greater than Even" 
  else if (evens > odds) return "Even is greater than Odd"
  else return "Even and Odd are the same"
}

File: team-5-definitely-colins-team/7-even-or-odd-which-is-greater.py
def even_or_odd(s):
    evens = 0
    odds = 0
    for digit in s: 
        d = int(digit)
        if d % 2 == 0: evens += d
        else: odds += d

    if odds > evens: return "Odd is greater than Even" 
    elif evens > odds: return "Even is greater than Odd"
    else: return "Even and Odd are the same"
